code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -385,7 +385,25 @@
 
 // Xenos eating fruit
 /obj/item/reagent_container/food/snacks/resin_fruit/attack(mob/living/carbon/xenomorph/affected_xeno, mob/user)
```

Comment:
The first argument is not correct (its not always a xeno being affected). Refactor back to `mob/` and correct the variable name (not always a xeno).